src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -465,6 +464,31 @@ public void testViewPhotoAndAjaxForLargeScaledSession() throws Exception {
         resultsPage.hoverClickAndViewStudentPhotoOnHeading("1-2", Const.SystemParams.DEFAULT_PROFILE_PICTURE_PATH);
     }
 
+    @Test
+    public void testViewNoSpecificSection() {
```

Comment:
Since there is no unit test for `bundle` `data` and `action` (that means we need to depend on the system test --- which is no a good practice). Pls make sure that the `InstructorFeedbackResultsPageUiTest.json` contains responses to/from instructors and students in no section.

Reply:
The Sessions I picked are chosen according to the responses it has. For example, `Sanitized Session` has no responses for `RGQ`. So `RGQ` has no `NO_SPECIFIC_SECTION` in that session, while `RGQ` will show `NO_SPECIFIC_SECTION` in `Open Session`.